src/main/java/com/toptrumps/core/engine/GameEventListener.java

```diff
@@ -12,6 +12,8 @@
     Attribute onRequestSelection(final Card card);
     void onAttributeSelected(final Player activePlayer);
     void onRoundEnd(final RoundOutcome outcome);
+    void onRoundEnd(final RoundOutcome outcome, final Player eliminatedPlayer);
+    void onRoundEnd(final RoundOutcome outcome, final ArrayList<Player> eliminatedPlayers);
     void onGameOver(Player winner);
-
+    void onAiRoundWin();
```

Comment:
We certainly don't need this! ```onRoundEnd``` can return the correct winner (doesn't matter if it's AI or not).